rust/core-lib/src/editor.rs

```diff
@@ -1058,6 +1058,17 @@ impl<W: Write + Send + 'static> Editor<W> {
         Some(text.slice_to_string(offset, end_off))
     }
 
+    pub fn plugin_get_selections(&self, view_id: &ViewIdentifier) -> Value {
+        //TODO: multiview support
```

Comment:
I definitely agree that it's important to maintain a distinction between the internal architecture and the API as exposed via plugin libraries; I've been trying to keep the plugin API (especially in the python lib) very view focused.

When I think about architecting this I come back to the asymmetry of the relationship between a `ViewIdentifier` and a `BufferIdentifier`; that is, that a `ViewIdentifier` implicitly also identifies a buffer; every view has a single constant buffer for the duration of its existence. 

Is there any advantage to thinking about some clever packed single identifier value?